readme.md

```diff
@@ -934,7 +934,7 @@ Default: [`Link` header logic](source/index.ts)
 
 The function takes three arguments:
 - `response` - The current response object.
-- `allItems` - An array of the emitted items.
+- `allItems` - An array of the emitted items if [pagination.stackAllItems](#pagination.stackAllItems) is set to `true`. An empty array otherwise.
```

Comment:
Can you move this property to the bottom of the list. Same in the types.

Can you make it clear that it's empty by default?

Reply:
Sure. Just for me to remember how to correctly update a branch from upstream... My git proficiency is rather basic.

Reply:
`git pull --rebase origin branch`

Reply:
Better?